File: Practica_1/src/gestorAplicacion/noPersonas/Jugo.java
package gestorAplicacion.noPersonas;

public class Jugo extends Bebida{
	
	private static int inventarioJugo;
	private static int codigoJugo = 2000;
	private static final float PRECIOJUGO = 4000;
	
	public Jugo() {
		super(codigoJugo, PRECIOJUGO);
		this.codigo = codigoJugo;
		this.precio = PRECIOJUGO;
		inventarioJugo ++;
	}
	
	public String toString() {
		String activo = "\nJugo";
		String formatoFactura = String.format("%-25s",activo).replace(' ', '.');
		formatoFactura += this.precio;
		return formatoFactura;
	}
	
	


	public static int getInventarioJugo() {
		return inventarioJugo;
	}


	public static void setInventarioJugo(int inventarioJugo) {
		Jugo.inventarioJugo = inventarioJugo;
	}
	
	public static void prepararJugos(int i) {
		while(i>0) {
			new Jugo();
			i--;
		}
	}
}
	

File: Practica_2/noPersonas/jugo.py
from noPersonas.bebida import Bebida

class Jugo(Bebida):
    _inventarioJugo = 9
    _codigoJugo = 2000
    _PRECIOJUGO = 4000
    
    def __init__(self):
        super().__init__(Jugo._codigoJugo, Jugo._PRECIOJUGO)
        self._codigo = Jugo._codigoJugo
        self._precio = Jugo._PRECIOJUGO
        Jugo._inventarioJugo -= 1
    
    def __str__(self):
        activo = '\Jugo'
        formatoFactura = ('%-25s')%activo
        formatoFactura = formatoFactura.replace(' ', '.')
        formatoFactura += str(self._precio)
        return formatoFactura    

    @classmethod
    def getInventarioJugo(cls):
        return cls._inventarioJugo

    @classmethod
    def setInventarioJugo(cls, inventarioJugo):
        cls._inventarioJugo = inventarioJugo
    
    @staticmethod
    def prepararJugos(i):
        while i > 0:
            Jugo()
            Jugo._inventarioJugo += 2
            i += -1